packages/lit-dev-content/samples/examples/motion-in-out/my-element.ts

```diff
@@ -0,0 +1,60 @@
+import {LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {animate, AnimateController, flyBelow, fade} from '@lit-labs/motion';
+import {styles} from './styles.js';
+
+@customElement('my-element')
+export class MyElement extends LitElement {
+  static styles = styles;
+
+  lit = ['L', 'I', 'T'];
```

Comment:
You need a pretty big monitor to see the whole Lit logo in this example. Maybe scale it proportionally to the window size?

Reply:
Done.